src/frontend/screens/SyncModal/SyncView.js

```diff
@@ -24,12 +24,12 @@ const Header = ({ onClosePress }: HeaderProps) => (
   </View>
 );
 
-const WifiBar = ({ onPress, ssid }) => (
+const WifiBar = ({ onPress, ssid, deviceName }) => (
   <TouchableNativeFeedback onPress={onPress}>
     <View style={styles.wifiBar}>
       <WifiIcon />
       <Text style={styles.wifiBarText}>
-        <Text style={styles.bold}>Wi-Fi:</Text> {ssid}
+        <Text style={styles.bold}>Wi-Fi:</Text> {ssid}, {deviceName}
```

Comment:
how do we want to display the device name?

Reply:
I think this is a good place to put it. Just tweaked formatting